addons/web/static/lib/hoot/core/expect.js

```diff
@@ -924,6 +925,58 @@ export class Matchers {
         });
     }
 
+    /**
+     * Expects the received string to be xml parsable and equal to the `expected` value upon
+     * formatting.
+     *
+     * @param {string} expected
+     * @param {ExpectOptions} [options]
+     * @example
+     *  expect("<div>A</div>").toBeXmlEqual(`
+     *      <div>
+     *          A
+     *      </div>
+     * `);
```

Comment:
These are not equivalent and [will render differently](https://odoo.github.io/owl/playground/#eyJqcyI6ImltcG9ydCB7IENvbXBvbmVudCwgbW91bnQgfSBmcm9tIFwiQG9kb28vb3dsXCI7XG5jbGFzcyBSb290IGV4dGVuZHMgQ29tcG9uZW50IHtcbiAgICBzdGF0aWMgdGVtcGxhdGUgPSBcIlJvb3RcIjtcbn1cbm1vdW50KFJvb3QsIGRvY3VtZW50LmJvZHksIHsgdGVtcGxhdGVzOiBURU1QTEFURVMsIGRldjogdHJ1ZSB9KTtcbiIsImNzcyI6Ii5ncmVldGVyIHtcbiAgICBmb250LXNpemU6IDIwcHg7XG4gICAgd2lkdGg6IDMwMHB4O1xuICAgIGhlaWdodDogMTAwcHg7XG4gICAgbWFyZ2luOiA1cHg7XG4gICAgdGV4dC1hbGlnbjogY2VudGVyO1xuICAgIGxpbmUtaGVpZ2h0OiAxMDBweDtcbiAgICBiYWNrZ3JvdW5kLWNvbG9yOiAjZWVlZWVlO1xuICAgIHVzZXItc2VsZWN0OiBub25lO1xufVxuIiwieG1sIjoiPHRlbXBsYXRlcz5cbiAgPHQgdC1uYW1lPVwiUm9vdFwiPlxuICAgIDxzcGFuID5BPC9zcGFuPlxuICAgIDxzcGFuIHN0eWxlPVwiYmFja2dyb3VuZDogYmxhY2s7IGNvbG9yOiB3aGl0ZVwiPlxuICAgICAgQVxuICAgIDwvc3Bhbj5cbiAgICA8c3BhbiA+QTwvc3Bhbj5cbiAgICA8c3BhbiBzdHlsZT1cImJhY2tncm91bmQ6IGJsYWNrOyBjb2xvcjogd2hpdGVcIj5cbiAgICAgIEFcbiAgICA8L3NwYW4+XG4gICAgPHNwYW4gPkE8L3NwYW4+XG4gICAgPHNwYW4gc3R5bGU9XCJiYWNrZ3JvdW5kOiBibGFjazsgY29sb3I6IHdoaXRlXCI+XG4gICAgICBBXG4gICAgPC9zcGFuPlxuICAgIDxzcGFuID5BPC9zcGFuPlxuICA8L3Q+XG48L3RlbXBsYXRlcz5cbiJ9) if the element is inline (notice the different width between black and white elements), not sure how important that may be when trying to assert xml equality